recipe/build-lib.sh

```diff
@@ -16,27 +16,27 @@ if [ ${cuda_compiler_version} != "None" ]; then
     DEPRECATED_IN_11=(35 50)
     if [ $(version2int $cuda_compiler_version) -ge $(version2int "11.1") ]; then
         # Ampere support for GeForce 30 (sm_86) needs cuda >= 11.1
-        ARCHES=( "${ARCHES[@]}" 75 80 86 )
+        ARCHES=( "${ARCHES[@]}" 75 80 )
         LATEST_ARCH=86
     elif [ $(version2int $cuda_compiler_version) -ge $(version2int "11.0") ]; then
         # Ampere support for A100 (sm_80) needs cuda >= 11.0
-        ARCHES=( "${ARCHES[@]}" 75 80 )
+        ARCHES=( "${ARCHES[@]}" 75 )
         LATEST_ARCH=80
     elif [ $(version2int $cuda_compiler_version) -ge $(version2int "10.0") ]; then
         # Turing support (sm_75) needs cuda >= 10.0
-        ARCHES=( "${DEPRECATED_IN_11[@]}" "${ARCHES[@]}" 75 )
+        ARCHES=( "${DEPRECATED_IN_11[@]}" "${ARCHES[@]}" )
```

Comment:
@dantegd not sure I get why the latest arch's are removed from the lists?

Reply:
I actually stumbled over the same point at first sight. But `LATEST_ARCH` is handled independently from `ARCHES`, and that's why there's no actual reduction in arch-support

Reply:
indeed, originally I got it wrong and the compilation string repeating the last arch, line 34 (inside the loop) adds all the archs with `real` (i.e. all except the last) and then line 39 adds the last arch without `real` or `virtual` (which makes it so it adds both virtual and real for the latest arch) and we end up with the desired

```bash
52-real;-60-real;61-real;70-real;75-real;80
```

(that was the 11.0 example)